app/routes/about-us.tsx

```diff
@@ -0,0 +1,7 @@
+export default function AboutUsRoute() {
+	return (
+		<main className="container py-16">
+			<h1 className="text-mega">About us</h1>
+		</main>
+	)
```

Comment:
This is excellent progress, Rich. When you finish, if you decide not to include these routes in your project either remove them or put some kind of 'Page currently under construction' message on them :)